utils/utils.go

```diff
@@ -403,3 +403,18 @@ func FindFileInDir(dirPath, fileName string) (string, error) {
 	}
 	return filePath, nil
 }
+
+// FixCommandWithApostrophe corrects the passed command which might have apostrophes in it.
+func FixCommandWithApostrophe(destDir *string) {
```

Comment:
Will `strings.Replace(destDir, "'", "''", -1)` not work?

Reply:
addressed